src/lambdapi.ml

```diff
@@ -279,6 +294,13 @@ let lsp_log_file : string Term.t =
   in
   Arg.(value & opt string default & info ["log-file"] ~docv:"FILE" ~doc)
 
+(** Specific to the ["decision-tree"] command. *)
+
+let qsym : string Term.t =
+  let doc = "Fully qualified symbol name with dot separated identifiers." in
+  Arg.(required & pos 0 (some string) None
+       & info [] ~docv:"MOD_PATH.SYM" ~doc)
```

Comment:
Please, rather use the same thing as everywhere else:
```ocaml
let i = Arg.(info [] ~docv:"MOD_PATH.SYM" ~doc) in
Arg.(required & pos 0 (some string) None i)
```

Reply:
Actually, you can even do the splitting of the module path here using `Cmcliner.Arg.list` (maybe combined with `Cmdliner.Arg.non_empty`).

Reply:
And the nicest thing would be that this function takes care of returning a pair of a module path and a symbol name. You can do that easily I think.

Reply:
To perform your last suggestion, I'd need a `map` function of type
`('a -> 'b) -> 'a conv -> 'b conv`, right?

Reply:
No, the full thing is a `string Term.t`, so assuming the current thing is called `arg`, you need to do something like `Term.(pure f $ arg)` where `f` is your function from `string list` to `Path.t * sym`.

Reply:
Nice.